okhttp/src/test/java/okhttp3/internal/tls/ClientAuthTest.java

```diff
@@ -265,6 +269,44 @@ public void setUp() {
     }
   }
 
+  @Test public void invalidClientAuthEvents() throws Throwable {
+    server.enqueue(new MockResponse().setBody("abc"));
+
+    clientCert = new HeldCertificate.Builder()
+        .signedBy(clientIntermediateCa)
+        .serialNumber(4L)
+        .commonName("Jethro Willis")
+        .addSubjectAlternativeName("jethrowillis.com")
```

Comment:
Is this a cultural reference? I don’t know it.

Reply:
I think it was a sliding doors Jesse became a country music star instead of a mild mannered software engineer reference back in the day

https://github.com/square/okhttp/blob/482f88300f78c3419b04379fc26c3683c10d6a9d/okhttp/src/test/java/okhttp3/internal/tls/ClientAuthTest.java#L112